charts/orchestrator/templates/_wait_for_crd.tpl

```diff
@@ -0,0 +1,84 @@
+{{- define "wait-for-crd-available" -}}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ printf "%s-crd-%s" .releaseName .resourceName }}
+  namespace: {{ .releaseNamespace }}
+  annotations:
+    "helm.sh/hook": post-install
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded,hook-failed
+    "helm.sh/hook-weight": "0"  
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ printf "%s-crd-%s" .releaseName .resourceName }}
+  annotations:
+    "helm.sh/hook": post-install
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded,hook-failed
+    "helm.sh/hook-weight": "0"
+rules:
+  - apiGroups: 
+    - apiextensions.k8s.io
+    resources:
+    - customresourcedefinitions
+    verbs:
+    - get 
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ printf "%s-crd-%s" .releaseName .resourceName }}
+  annotations:
+    "helm.sh/hook": post-install
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded,hook-failed
+    "helm.sh/hook-weight": "0"
+subjects:
+  - kind: ServiceAccount
+    name: {{ printf "%s-crd-%s" .releaseName .resourceName }}
+    namespace: {{ .releaseNamespace }}
+roleRef:
+  kind: ClusterRole
+  name: {{ printf "%s-crd-%s" .releaseName .resourceName }}
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ printf "%s-crd-%s" .releaseName .resourceName }}
+  namespace: {{ .releaseNamespace }}
+  annotations:
+    "helm.sh/hook": post-install
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded # add ,hook-failed once verified
+    "helm.sh/hook-weight": "1"    
+spec:
+  template:
+    metadata:
+      name: {{ printf "%s-crd-%s" .releaseName .resourceName }}
+    spec:
+      serviceAccountName: {{ printf "%s-crd-%s" .releaseName .resourceName }}
+      restartPolicy: Never
+      containers:
+      - name: deploy-manifest
+        image: registry.redhat.io/openshift4/ose-cli:latest
+        command:
+          - "bin/bash"
+          - "-c"
+        args:
+          - |
+            echo "Wait for condition"
```

Comment:
I wanted to suggest including also the CRD name in this comment for debugging purposes, but I think that with adding `hook-failed`, there is nothing else to monitor.

So thinking out loud how to recommend users to debug the installation process:
I think we can ask them to watch for this job's log in case of a failure by re-installing.
This can be added to the NOTES.txt. WDYT?

Reply:
Sure. I'll mention it in the `NOTES.txt`